### DISCUSSION TITLE
SUGGESTION: Station and Warehouse (no flow notification)

### DISCUSSION THREAD

**Discussion by JezzyBelle** [voxeltycoon/issues#843] [General]
Like we have unprofitable vehicle notices (which I also think should have more configurability), I would like to see an ability to set a station or warehouse with a warning that shows no product has moved in/out for X time period. That would quickly show me where I have no flow, or a vehicle that has been trying to pick up wrong product causing no pickup. Key here is that it would have to be configurable (on/off, time period, etc)  TBH, for me, this would be way more desirable for my play style than the vehicle (no product for 30 days notice). I tend to shut that off on all vehicles anyway as it can become annoying. The common denominator in most instance are the stations and warehouses which cause those vehicles to generate the 30 day notice. One station backed up can cause large numbers of notices, more than one issue, general chaos to the point where notices seem spammy..... When it is just a station or warehouse, it targets the issue which is impacting all those vehicles more directly making it more manageable.

**Comment by kalmix88-hun** [comment]
Not a bad idea. I would like to see the flow indicator in one of the researches.
